BlueprintUI/Sources/Layout/Row.swift

```diff
@@ -39,7 +39,7 @@ public struct Row: StackElement {
             }
         }
 
-        var stackAlignment: StackLayout.Alignment {
+        public var stackAlignment: StackLayout.Alignment {
```

Comment:
Intentional? It's not public on `ColumnAlignment`.

Reply:
I needed this at one point, but seems like it is no longer necessary. Good catch